packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -47,8 +47,7 @@ const propTypes = {
 
 const defaultProps = {
   text: '',
-  // TODO: textStyle should be set to 'primary' on the next major version bump: https://github.com/cerner/terra-clinical/issues/526
-  textStyle: undefined,
+  textStyle: 'primary',
```

Comment:
How would a user revert back to the previous default / un-styled text? Or is our intention with this change to discourage any use of un-styled text?

For example, a user with a component prior to this change that looks like this would display with the default un-styled text.

```jsx
<ItemDisplay text="Example Display" />
```

This example after this change would update to primary text. To revert back I believe a user would need to do something like this which feels a little strange:

```jsx
<ItemDisplay text="Example Display" textStyle='' />
```


cc: @neilpfeiffer Do we want to discourage any use of the ItemDisplay with un-styled text, do we want to provide a new textStyle?

Reply:
It would be my opinion that we would not want a consumer using unstyled text anywhere.

Reply:
It technically doesn't matter (if you only examine this from the prop standpoint, it seems like a big deal but in actuality it isn't) -- since the style of `undefined` is/was/has-been exactly the same as `primary`--> this is mainly a fix to conceptualization of the options and clearer/cleaner with documenting in the propsTable (after change will show the default is 'primary') which then matches the examples above, and makes more sense when we spell out the options 'Primary', 'Secondary', 'Attention', etc. 

It was odd when there was a `'secondary'` without a `'primary'` (undefined has always equaled primary, and was called out in css classes and examples, but not as prop), so it was added, and now it is odd having undefined and `'primary'` both doing the same thing. This MVB will not visually/wdio affect anyone who has currently not set a `textStyle`, and won't affect anyone who has - it simply is for better documentation and understanding of the text options available (which might also eventually get put into other components).

To @jeremyfuksa's point - there really is no reason to unset this, because unsetting is the same as setting it to primary text (our default styling). Nor would 'undefined' ever look differently and need a new/another style.

And to echo @StephenEsser -- `textStyle=''` is just strange (...though I was thinking of another word).